azurerm/resource_arm_stream_analytics_reference_input_blob.go

```diff
@@ -0,0 +1,243 @@
+package azurerm
```

Comment:
This code is pretty much an exact replica of the `..._stream_analytics_**stream**_input_blob.go`.